php/EE/Migration/Executor.php

```diff
@@ -2,28 +2,26 @@
 
 namespace EE\Migration;
 
-use \EE;
-use \EE\Model\Migration;
-use \EE\Utils;
-use Symfony\Component\Finder\Finder;
+use EE;
+use EE\Model\Migration;
+use EE\Utils;
 
 class Executor {
 
-    const MIGRATION_PATH = EE_ROOT . '/migrations';
+    const MIGRATION_PATH = EE_ROOT . '/migrations/';
 
     /**
      * Executes all pending migrations
      */
     public static function execute_migrations() {
 
         Utils\delem_log( "ee migration start" );
-        EE::log( "Migrating EasyEngine data to new version" );
+        EE::log( "Executing migrations" );
 
         $migrations = self::get_migrations_to_execute();
 
         if( empty( $migrations ) ) {
             EE::success( "Noting to migrate" );
-            exit( 0 );
```

Comment:
Add `return` statement here. No need to take execution further if `$migrations` are empty.